README.md

```diff
@@ -101,6 +101,31 @@ To access the your deployment on Gitpod, you will have to confirm the message th
 gp ports list
 ```
 
+## Contributing
+
+Before creating a pull request, test a final time and check for errors:
+
+`yarn test:with-build`
+`yarn check:format`
+`yarn format:fix`
+`yarn check:all`
```

Comment:
You could really just have `yarn check:all` here, since it checks formatting, linting, types, build, and test. The individual things you can run to fix these issues is probably good to share.

Reply:
~~~
Before creating a pull request, test a final time and check for errors:

```sh
yarn check:all
```

We provide a couple of scripts to automatically fix linting and formatting issues, where possible:

```sh
yarn lint:fix
yarn format:fix
```
~~~

Reply:
Given that this section is entitled "Contributing", we probably also want to [add a `CONTRIBUTING.md` file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors) at some point but that can be done outside of this PR.